config/routes.rb

```diff
@@ -7,6 +7,7 @@
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
   get '/dashboard', to: 'users#show'
+  get '/profile/:id', to: 'users#profile'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.